xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetable.css

```diff
@@ -179,13 +169,6 @@ thead.xwiki-livetable-display-header td.xwiki-livetable-display-header-filter se
 tbody.xwiki-livetable-display-body tr {
   border-bottom: 1px solid $theme.borderColor;
 }
-tbody.xwiki-livetable-display-body tr:hover , tbody.xwiki-livetable-display-body tr.rowHover { /* for IE6 tr:hover fix*/
```

Comment:
We should also put back "thead.xwiki-livetable-display-header tr.xwiki-livetable-display-filters:hover" too, sorry for missing these. I've pushed a fix to my branch. (I thought you had fixed both, then realized that you had only fixed the one.)